src/renderers/dom/fiber/ReactDOMFiberComponent.js

```diff
@@ -65,6 +65,14 @@ var {
 } = DOMNamespaces;
 
 if (__DEV__) {
+  var warnedUnknownTags = {
+    // People often believe <time> is a part of HTML5, but it was dropped.
+    // However it is currently widely used for semantic purposes, and Chrome
+    // intends to ship it (as of July 2017). To avoid flooding people with warnings,
+    // we intentionally *don't* warn about <time> even if it's unrecognized.
+    time: true,
```

Comment:
Hmm. I guess I misunderstood.

It was deleted but then re-added.

https://www.wired.com/2012/02/the-html5-time-element-is-back-and-better-than-ever/